A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/SAI/A320_Neo_SAI.js

```diff
@@ -1024,16 +1110,16 @@ class A320_Neo_SAI_SelfTestTimer extends HTMLElement {
         this.selfTestDiv.setAttribute("height", "100%");
         this.appendChild(this.selfTestDiv);
 
-        this.selfTestSVG = document.createElementNS(Avionics.SVG.NS, "svg");
-        this.selfTestSVG.setAttribute("id", "SelfTestSVG");
-        this.selfTestSVG.setAttribute("viewBox", "0 0 600 600");
-        this.selfTestSVG.style.position = "absolute";
-        this.selfTestSVG.style.top = "0%";
-        this.selfTestSVG.style.left = "0%";
-        this.selfTestSVG.style.width = "100%";
-        this.selfTestSVG.style.height = "100%";
-        this.selfTestSVG.style.backgroundColor = "rgba(13,20,35,1)";
-        this.selfTestDiv.appendChild(this.selfTestSVG);
+        const selfTestSVG = document.createElementNS(Avionics.SVG.NS, "svg");
```

Comment:
Use let here, while the actual reference isn't changed, some properties are. Just better use of const and let